rust/src/detect/byte_math.rs

```diff
@@ -491,8 +453,18 @@ mod tests {
     }
 
     fn valid_test(
-        args: &str, nbytes: u8, offset: i32, oper: ByteMathOperator, rvalue_str: &str, nbytes_str: &str, rvalue: u32,
```

Comment:
rustfmt does not agree with this change 🤔

Reply:
@catenacyber I reran `rustfmt` and it yielded the same results. What are you seeing that shows `rustfmt` doesn't like the change?

Reply:
```diff
diff --git a/rust/src/detect/byte_extract.rs b/rust/src/detect/byte_extract.rs
index 6839d19c57..51bd6b78ef 100644
--- a/rust/src/detect/byte_extract.rs
+++ b/rust/src/detect/byte_extract.rs
@@ -319,15 +319,8 @@ mod tests {
     }
 
     fn valid_test(
-        args: &str,
-        nbytes: u8,
-        offset: i16,
-        var_name_str: &str,
-        base: ByteBase,
-        endian: ByteEndian,
-        align_value: u8,
-        multiplier_value: u16,
-        flags: u16,
+        args: &str, nbytes: u8, offset: i16, var_name_str: &str, base: ByteBase,
+        endian: ByteEndian, align_value: u8, multiplier_value: u16, flags: u16,
     ) {
         let bed = SCDetectByteExtractData {
             nbytes,
diff --git a/rust/src/detect/byte_math.rs b/rust/src/detect/byte_math.rs
index 2b29c5b26e..4d489762f2 100644
--- a/rust/src/detect/byte_math.rs
+++ b/rust/src/detect/byte_math.rs
@@ -453,18 +453,9 @@ mod tests {
     }
 
     fn valid_test(
-        args: &str,
-        nbytes: u8,
-        offset: i32,
-        oper: ByteMathOperator,
-        rvalue_str: &str,
-        nbytes_str: &str,
-        rvalue: u32,
-        result: &str,
-        base: ByteBase,
-        endian: ByteEndian,
-        bitmask_val: u32,
-        flags: u8,
+        args: &str, nbytes: u8, offset: i32, oper: ByteMathOperator, rvalue_str: &str,
+        nbytes_str: &str, rvalue: u32, result: &str, base: ByteBase, endian: ByteEndian,
+        bitmask_val: u32, flags: u8,
     ) {
         let bmd = DetectByteMathData {
             nbytes,
```

Reply:
With rustfmt 1.7.0-nightly (1388d7a0 2024-03-20)

Reply:
We should probably use latest Rust stable for fmt checks.

Reply:
@catenacyber Thanks ... I was using an older version of `rustfmt` -- 1.5